### PR TITLE
Add pull-request template 🙃

### PR DESCRIPTION
**Pull request by vdemeester**
This adds two pull-request template.
- free form (for any type of pull-request)
- tep for TEPs, that would set the correct kind directly

We can go further and have a template for adding a member to the org, …

/cc @afrittoli @bobcatfish 

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

### CODE REVIEW

**Review by dibyom — COMMENTED** [review]
/lgtm

File: .github/PULL_REQUEST_TEMPLATE/tep.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+about: Create a TEP pull-request
+---
+
+<!-- 🎉🎉🎉 Thank you for the PR!!! 🎉🎉🎉 -->
+
+# Summary
+
+<!--
+Summarize your TEP here - ideally you can get that straight from your
+descriptive commit message or the summary of your TEP.
```
Comment (bobcatfish) [inline .github/PULL_REQUEST_TEMPLATE/tep.md:12]: would it make sense to include a link to our TEP docs?

**Review by bobcatfish — COMMENTED** [review]
Thanks @vdemeester !!

/approve

File: .github/PULL_REQUEST_TEMPLATE/tep.md (hunk 1 above)
Comment (vdemeester) [reply .github/PULL_REQUEST_TEMPLATE/tep.md:12]: Indeed 🙃

**Pull request merged**